src/Illuminate/Support/ServiceProvider.php

```diff
@@ -134,6 +157,78 @@ protected function publishes(array $paths, $group = null)
     }
 
     /**
+     * Register a view path to be published to the app's configured view directory.
+     *
+     * @param $path
+     * @param $namespace
+     * @param null $group
+     */
+    protected function publishesViews($path, $namespace, $group = null)
+    {
+        $viewPaths = $this->app['config']->get('view.paths', [resource_path('views/vendor/')]);
+        $this->publishes([
+            $path => rtrim($viewPaths[0], '/')."/vendor/$namespace",
```

Comment:
It's as close to the current behavior as possible. If you have the default config, this will just publish to that path. If you have multiple paths installed, it doesn't make sense to publish to all of them, so I figured publishing to the first makes the most sense.